packages/victory-bar/src/victory-bar.tsx

```diff
@@ -141,7 +141,12 @@ class VictoryBarBase extends React.Component<VictoryBarProps> {
       return this.animateComponent(props, animationWhitelist);
     }
 
-    const children = this.renderData(props);
+    let children;
+    if (props.groupComponent.type.displayName === 'VictoryClipContainer') {
+      children = this.renderData(props, () => true);
```

Comment:
This needs some improvement too. `renderData` has incorrect typing right now (it's type only accepts the `props` parameter), and that always true function can be moved out to a static class property or something else named.